### PR TITLE
Farm cards new UI expanded

### PR DESCRIPTION
**Pull request by memacius32**
Adding Expanded section content:

- Claimable Rewards.
- User deposits.
- Pool Type and Total staked (in mobile)
- Manage farm box

Note: 
Will Remove Manage page in a new pr (as it's now unused)

### Preview: 
Desktop:
Screen Shot 2022-09-14 at 01 06 48
Mobile:
Screen Shot 2022-09-14 at 01 06 04

#### Video:

https://user-images.githubusercontent.com/96993065/190025315-ef5ab212-9826-410c-9558-def10955db1b.mp4

### CODE REVIEW

**Comment by specialist0x** [comment]
Can you address the `50%` `max` buttons, it looks like one is selected when it shouldn't be:

image

File: src/components/Toggle/index.tsx [hunk 1]
```diff
@@ -3,15 +3,15 @@ import styled from 'styled-components'
...
+const ToggleElement = styled.span<{ isActive?: boolean; isOnSwitch?: boolean; fontSize?: string; padding?: string }>`
   padding: 0.25rem 0.5rem;
   border-radius: 14px;
   background: ${({ theme, isActive, isOnSwitch }) => (isActive ? (isOnSwitch ? theme.primary1 : theme.text4) : 'none')};
   color: ${({ theme, isActive, isOnSwitch }) => (isActive ? (isOnSwitch ? theme.white : theme.text2) : theme.text3)};
   font-size: 1rem;
   font-weight: 400;
 
-  padding: 0.35rem 0.6rem;
+  padding: ${({ padding }) => (padding ? padding : '0.35rem 0.6rem')};
```
Comment (specialist0x) [inline src/components/Toggle/index.tsx:14]:
```suggestion
  padding: ${({ padding }) => (padding ?? '0.35rem 0.6rem')};
```

File: src/components/earn/PoolCardTri/ManageStake.tsx [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+      toggleWalletModal()
+    }
+  }, [account, toggleWalletModal])
+
+  function handleDepositClick(event: React.MouseEvent) {
+    enableDepositModal()
+    event.stopPropagation()
+  }
+
+  function handleWithDrawClick(event: React.MouseEvent) {
```
Comment (specialist0x) [inline src/components/earn/PoolCardTri/ManageStake.tsx:78]:
```suggestion
  function handleWithdrawClick(event: React.MouseEvent) {
```

File: src/components/earn/PoolCardTri/PoolCardTri.styles.tsx [hunk 3]
```diff
@@ -103,26 +108,30 @@ export const StyledPairContainer = styled(PairContainer)`
...
   justify-items: center;
+
   ${({ theme }) => theme.mediaWidth.upToSmall`
-    grid-template-columns: 220px auto auto 80px;
- `};
+    grid-template-columns: 1fr 0.5fr 92px;
+  `};
   ${({ theme }) => theme.mediaWidth.upToExtraSmall`
-    grid-template-columns: 180px  auto 85px;
+    grid-template-columns: 167.5px  auto 92px;
```
Comment (specialist0x) [inline src/components/earn/PoolCardTri/PoolCardTri.styles.tsx:120]: I don't think you can have pixels as decimals

File: src/components/earn/PoolCardTri/index.tsx [hunk 4]
```diff
@@ -93,18 +116,18 @@ const DefaultPoolCardtri = ({
...
+  const currenciesQty = currencies.length
+  const farmName = friendlyFarmName ?? currencies.map(({ symbol }) => symbol).join('-')
+
   function onCardClick() {
     setShowMore(!showMore)
   }
 
   function handleClaimClick(event: React.MouseEvent) {
-    enableModal()
+    enableClaimModal()
```
Comment (specialist0x) [inline src/components/earn/PoolCardTri/index.tsx:127]: Should `event.stopPropagation()` be invoked first?  Might cause a race condition

**Review by specialist0x — APPROVED** [review]
Accepting to unblock, please make suggested changes

File: src/components/earn/PoolCardTri/index.tsx (hunk 4 above)
Comment (memacius32) [reply src/components/earn/PoolCardTri/index.tsx:127]: Changed!

**Comment by memacius32** [comment]
> Can you address the `50%` `max` buttons, it looks like one is selected when it shouldn't be:
> 
> image

Good catch, fixed 🙏 
Screen Shot 2022-09-21 at 15 44 40

Addressed rest of coments and Merging

**Pull request merged**